pandas/io/formats/format.py

```diff
@@ -943,6 +943,11 @@ def get_col_type(dtype):
                          if x else '{}') for x in row]
             else:
                 crow = [x if x else '{}' for x in row]
+            if self.fmt.kwds.get('bold_rows', True) and self.fmt.index:
```

Comment:
in the constructor do a ``self.bold_rows = self.fmt.kwd.get('bold_rows', True)``; see ``HTMLFormatter``. 

Then just use this here